### ISSUE TITLE
Develop robust alternative to abnormal_yn and result_in_range_yn

### ISSUE THREAD

**Issue by SanthoshBala** [HealthRex/CDSS#68]
**tl;dr - gonna start using result_flag instead of result_in_range, but it's still imperfect.**

To date, we've been using a combination of abnormal_yn and result_in_range_yn to determine whether a lab test's results were in a normal range.

However, both of these fields are unreliable.

**result_in_range_yn often has NULL values.** 
It seems like some lab tests get these values filled in, while others do not. 
Exactly 3 of the tens of thousands of tests have result_in_range_yn=N, and very few components seem to ever have result_in_range_yn=Y.

For example, of all LABA1C lab panels in the database, ~23k are abnormal_yn=NULL. Half result in an abnormal_yn=Y panel, and half result in abnormal_yn=NULL.

**result_flag is a stronger, though imperfect signal**
For example, 100% of the abnormal_yn=Y LABA1C and LABLAC panels have components with result_flag=High.

It's imperfect because different components seem to use different (and confusing) values for result_flag: High, Low, (None), High Panic, Low Panic, Low Off-Scale, Negative, Positive, Resistant, Susceptible, Abnormal, and *. They're not mutually exclusive (i.e. sometimes the same numeric value can get mapped to multiple flags.

**abnormal_yn doesn't seem to be consistently based on any other field**
result_flag seems like the best bet, but it's not idea.
